### ISSUE TITLE
Testing: Add missing accessible helper test to calcite-block-section

### ISSUE THREAD

**Issue by jcfranco** [Esri/calcite-app-components#552]
Stems from https://github.com/Esri/calcite-app-components/issues/385

**Comment by jcfranco** [comment]
PR depends on `calcite-components@v1.0.0-beta.16` (when it's released).

**Comment by jcfranco** [comment]
Installed. @kat10140 Can you help verify this one?

**Comment by kat10140** [comment]
Lewks gewd!
